### PR TITLE
pass single pass type to query params if it exists

### PR DESCRIPTION
**Pull request by erwinmombay**
this allows me to group the single pass experiment easier for errors

### CODE REVIEW

File: src/error.js [hunk 1]
```diff
@@ -499,6 +499,16 @@ export function getErrorReportData(message, filename, line, col, error,
   } else if (getMode().runtime) {
     runtime = getMode().runtime;
   }
+
+  const singlePassType = getMode().singlePassType;
+  if (singlePassType) {
+    if (singlePassType === 'sp') {
```
[the remark this answers is not in the document]
Comment (erwinmombay) [reply src/error.js]: @prateekbh yes. it most likely does

[the remark this answers is not in the document]
Comment (erwinmombay) [reply src/error.js]: updated PTAL

File: src/error.js [hunk 2]
```diff
@@ -499,6 +499,12 @@ export function getErrorReportData(message, filename, line, col, error,
   } else if (getMode().runtime) {
     runtime = getMode().runtime;
   }
+
+  const singlePassType = getMode().singlePassType;
+  if (['sp', 'mp', 'esm'].includes(singlePassType)) {
```
Comment (jridgewell) [inline src/error.js]: We'll need to update server, too.

Comment (erwinmombay) [reply src/error.js]: will do

File: src/service/extension-location.js [hunk 3]
```diff
@@ -63,7 +72,8 @@ export function calculateExtensionScriptUrl(location, extensionId,
   const extensionVersion = opt_extensionVersion
     ? '-' + opt_extensionVersion
     : '';
-  return `${base}/rtv/${rtv}/v0/${extensionId}${extensionVersion}.js`;
+  return `${base}/rtv/${rtv}/v0/${getSinglePassExperimentPath()}` +
```
Comment (erwinmombay) [inline src/service/extension-location.js]: @jridgewell PTAL. i've added this code for the script url calculation

**Comment by erwinmombay** [comment]
@jridgewell done. removed the fix for the script calc, and submitted new PR

**Pull request merged**